frontend/integration-tests/tests/kubevirt/template.wizard.scenario.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-undef, max-nested-callbacks */
+import { execSync } from 'child_process';
+import { OrderedMap } from 'immutable';
+import { browser, ExpectedConditions as until } from 'protractor';
+
+// eslint-disable-next-line no-unused-vars
+import { networkResource, provision as provisionOptions, removeLeakedResources, storageResource, VM_BOOTUP_TIMEOUT } from './utils';
+import { appHost, testName } from '../../protractor.conf';
+import { deleteRow, errorMessage, filterForName, isLoaded, resourceRowsPresent } from '../../views/crud.view';
+import { basicVmConfig, glusterfsDisk, hddDisk, networkInterface, testNAD } from './mocks';
+import * as wizardView from '../../views/kubevirt/wizard.view';
+import Wizard from './models/wizard';
+
+describe('Kubevirt create VM template using wizard', () => {
+  const leakedResources = new Set<string>();
+  const wizard = new Wizard();
+  const provisionConfigs = OrderedMap<string, {provision: provisionOptions, networkOptions: networkResource, storageOptions: storageResource}>()
+    .set('URL (+2 disks, +1 NIC)', {
+      provision: {
+        method: 'URL',
+        source: basicVmConfig.sourceURL,
+      },
+      networkOptions: [networkInterface],
+      storageOptions: [hddDisk, glusterfsDisk],
+    })
+    .set('Container (+2 disks, +1 NIC)', {
+      provision: {
+        method: 'Container',
+        source: basicVmConfig.sourceContainer,
+      },
+      networkOptions: [networkInterface],
+      storageOptions: [glusterfsDisk, hddDisk],
+    })
+    .set('PXE (+2 disks, +1 NIC)', {
+      provision: {method: 'PXE'},
+      networkOptions: [networkInterface],
+      storageOptions: [glusterfsDisk, hddDisk],
+    });
+
+  beforeAll(async() => {
+    execSync(`echo '${JSON.stringify(testNAD)}' | kubectl create -f -`);
+  });
+
+  afterAll(async() => {
+    execSync(`kubectl delete -n ${testName} net-attach-def ${testNAD.metadata.name}`);
+    removeLeakedResources(leakedResources);
```

Comment:
```removeLeakedResources(leakedResources);``` should move to afterEach block because if some test case in this describe block fails, other will too due to resources.

Reply:
Updated.